quarkus-extension/engine/deployment/src/test/resources/org/camunda/bpm/quarkus/engine/test/config/mixed-application.properties

```diff
@@ -2,7 +2,6 @@ quarkus.camunda.cmmn-enabled=false
 quarkus.camunda.dmn-enabled=false
 quarkus.camunda.history=none
 quarkus.camunda.enforce-history-time-to-live=false
-quarkus.camunda.initialize-telemetry=false
```

Comment:
❓ I assume that the `quarkus-UNIT` stage currently fails because the getter for this property was removed, but the property is still used. Is that correct?

Reply:
Yes, correct.